app/grandchallenge/challenges/views.py

```diff
@@ -389,3 +399,158 @@ def form_valid(self, form):
         response = HttpResponse()
         response["HX-Refresh"] = "true"
         return response
+
+
+class ChallengeCostViewPermissionMixin(UserPassesTestMixin):
+    def test_func(self):
+        return self.request.user.is_staff or self.request.user.has_perm(
```

Comment:
I think we should just work with the `challenges.view_challengerequest` permission, then the Mixin can be removed too.